apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -61,17 +61,16 @@ export function ElectionScreen(): JSX.Element {
           </P>
         </div>
       </ElectionCard>
-      <div style={{ display: 'flex', gap: '1rem' }}>
-        {(isSystemAdministratorAuth(auth) || isElectionManagerAuth(auth)) && (
+      {isSystemAdministratorAuth(auth) && (
+        <div style={{ display: 'flex', gap: '1rem' }}>
           <ExportElectionPackageModalButton />
-        )}
-        {isSystemAdministratorAuth(auth) && (
           <UnconfigureMachineButton
             isMachineConfigured
             unconfigureMachine={unconfigureMachine}
           />
-        )}
-      </div>
+        </div>
+      )}
+      {isElectionManagerAuth(auth) && <ExportElectionPackageModalButton />}
```

Comment:
Made a small code reorg to have it be rendering buttons by user type. Either way one line of code was duplicated (`isSystemAdmin type check` or `ExportElectionPackageModalButton`). This way might be a bit easier to read, but it's inconsequential.

Reply:
Given that this screen will only ever be rendered for system admins or election managers, it should be safe to just check for the sys admin auth to conditionally render the unconfigure button.

If we wanted to be extra sure about that assumption, we could add something like this at the top of the screen component:
```ts
assert(isSystemAdministrator(auth) || isElectionManager(auth))
```

Reply:
Ah that makes complete sense given that this screen is only rendered for those two user types. Will keep that in mind going forward.

And that `assert` makes sense to provide the certainty - in reality do we want to add that check here or was it more of an instructional thought?

Reply:
I'm in favor of adding the `assert` to be defensive, though the stakes feel fairly low here so I wouldn't insist on it.